media-libs/opencv/opencv-3.4.0.recipe

```diff
@@ -249,6 +246,8 @@ INSTALL()
 
 TEST()
 {
+	# Fails as it needs the main packages, but they won't gets attached to the chroot
```

Comment:
I guess the tests need the shared libs.
Well, you can workaround this by setting LIBRARY_PATH to an appropriate value.
Assuming the libs are available in src/lib/ after the build, you can do this:
```
TEST()
{
	LIBRARY_PATH="$sourceDir/src/lib${LIBRARY_PATH:+:$LIBRARY_PATH}" \
	make -C build_haiku test
}
```
(Don't forget to replace src/lib by the relative path to the real directory containing the libs.)